### ISSUE TITLE
Подсветка кода в статьях

### ISSUE THREAD

**Issue by vitalybaev** [web-standards/web-standards.ru#92]
Как мне кажется, будет удобнее иметь полноценную подсветку кода в блоках в статьях.
Веса много не добавит, подключается тоже достаточно просто.
Необходимо разве что зафиксировать цвета.

**Comment by pepelsbey** [comment]
С подстветкой главная проблема в темах: они обычно стандартные и стрёмные, мимо дизайна сайта. Я бы начал с подбора темы, которая может лечь на наши блоки кода в градиентах.

**Comment by akellbl4** [comment]
Взял на себя смелость примерить цветовую палитру для подсветки синтаксиса.
Использовал официальный плагин [@11ty/eleventy-plugin-syntaxhighlight](https://www.11ty.dev/docs/plugins/syntaxhighlight/) c темой [Coy](https://prismjs.com/download.html#themes=prism-coy&languages=markup+css+clike+javascript). В целом я думаю цвета еще можно докрутить.
<details>
<summary>Скриншоты</summary>

![CleanShot 2020-10-18 at 23 11 21](https://user-images.githubusercontent.com/2330682/96378866-b0455600-1197-11eb-9a74-570fd8c144f7.png)
![CleanShot 2020-10-18 at 23 11 11](https://user-images.githubusercontent.com/2330682/96378870-b804fa80-1197-11eb-96c3-404679fe5dae.png)
![CleanShot 2020-10-18 at 23 11 01](https://user-images.githubusercontent.com/2330682/96378871-b9362780-1197-11eb-8da1-b9e9713c4b59.png)
![CleanShot 2020-10-18 at 23 10 30](https://user-images.githubusercontent.com/2330682/96378872-baffeb00-1197-11eb-81e3-b9616a5227a1.png)
</details>

**Comment by pepelsbey** [comment]
Тут, к сожалению, очень сильно просаживается контрастность цветов. Одним из требованием к любой теме должно быть, чтобы контрастность текста к фону не была ниже 4.5.

**Comment by akellbl4** [comment]
Да, при неоднородном и достаточно ярком фоне в любом случае придётся докручивать цвета.

**Comment by higimo** [comment]
@pepelsbey я правильно понимаю, тут недостаточный контраст?
![image](https://user-images.githubusercontent.com/2804205/104790102-a8ea1480-57a7-11eb-99d8-7d9cc05c60fe.png)
